frontend/src/widgets/QualityAssuranceDashboardOverview.scss

```diff
@@ -0,0 +1,47 @@
+@media( min-width: 640px){
```

Comment:
would it be possible to reuse the HTML/CSS from the existing widgets and just override stuff here where necessary, rather than recoding this?